### PR TITLE
Fix the revision of aos

### PR DESCRIPTION
**Pull request by iusyk**
scope
aos-servicemanager
aos-updatemanager

Signed-off-by: Ihor Usyk <ihor_usyk@epam.com>

### CODE REVIEW

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aos_addons/aos-addons.bb [hunk 1]
```diff
@@ -7,6 +7,8 @@ LICENSE = "CLOSED"
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRC_URI = "git://git@gitpct.epam.com/epmd-aepr/aos_servicemanager.git;protocol=ssh;branch=master;tag=demo_0.1"
+SRCREV = "566325a2d07d63d44433a9313e4030f28b63bd42"
```
Comment (al1img) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aos_addons/aos-addons.bb]: It looks like it already fixed to tag=demo_0.1. So I guess this PR not needed.

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aosservicemanager/aosservicemanager_git.bb [hunk 2]
```diff
@@ -6,6 +6,7 @@ GO_IMPORT = "gitpct.epam.com/epmd-aepr/aos_servicemanager"
 
 SRCREV = "demo_0.1"
 SRC_URI = "git://git@${GO_IMPORT}.git;protocol=ssh;destsuffix=${PN}-${PV}/src/${GO_IMPORT}"
+SRCREV = "566325a2d07d63d44433a9313e4030f28b63bd42"
```
Comment (al1img) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aosservicemanager/aosservicemanager_git.bb]: Same as above.

**Comment by iusyk** [comment]
not relevant

**Comment by iusyk** [comment]
no changes

**Pull request closed without merging**